final-project/static/scripts.js

```diff
@@ -21,10 +34,18 @@ function editProduct()
 
 
 
-/**
- * Updates product search and/or product specific changes.
- */
-function update()
-{
+// Updates product search and/or product specific changes.
+function update() {
+
+  // TODO: Get element I want to change from html
+    // https://learn.jquery.com/using-jquery-core/jquery-object/#the-jquery-object
+
+  // TODO: Use the .map() method to go through each object on the product page
```

Comment:
You can actually map one format into another. This can help you extract or modify content.